pkg/runs/submitter.go

```diff
@@ -465,13 +465,15 @@ func (submitter *Submitter) isRasDetailNeededForReports(params utils.RunsSubmitC
 
 func (submitter *Submitter) buildListOfRunsToSubmit(portfolio *Portfolio, runOverrides map[string]string) []TestRun {
 	readyRuns := make([]TestRun, 0, len(portfolio.Classes))
-
+	currentUser , _ := user.Current()
```

Comment:
As discussed, we can use the `GetCurrentUserName()` method here instead since that performs some error handling.